SECURITY.md

```diff
@@ -0,0 +1,74 @@
+# Security Policy
+
+## Supported Versions
+
+Unless otherwise specified, we recommend to only use the most recent minor version release.
+
+## Reporting a Vulnerability
+
+**Please do not file a public ticket** mentioning the vulnerability.
+
+To disclose a vulnerability, please submit a `Security Advisory` via the `Security` tab on the impacted repository.
+
+If a repository doesn't have the proper security reporting set up, please email `security@celestia.org` to report the vulnerability.
+
+## Disclosure Policy
+
+Vulnerability details may be shared with third parties after the vulnerability
+has been fixed and the program owner has provided permission to disclose or
+after 120 days from submission, whichever is sooner.
+
+### Scope
+
+The scope of this security policy applies to the code repositories under the [Celestia Org Github](https://github.com/celestiaorg) and any related infrastructure.
```

Comment:
I wonder if we should include something about how we handle forks. Of which we have quite a few in our org 🤔